content/_code-samples/quickstart/js/8.escrow.html

```diff
@@ -0,0 +1,305 @@
+<html>
+  <head>
+    <title>Time-based Escrow Test Harness</title>
+    <link href='https://fonts.googleapis.com/css?family=Work Sans' rel='stylesheet'>
+    <style>
+       body{font-family: "Work Sans", sans-serif;padding: 20px;background: #fafafa;}
+       h1{font-weight: bold;}
+       input, button {padding: 6px;margin-bottom: 8px;}
+       button{font-weight: bold;font-family: "Work Sans", sans-serif;}
+       td{vertical-align: middle;}
+    </style>
+    <script src='https://unpkg.com/xrpl@2.7.0/build/xrpl-latest-min.js'></script>
+    <script src='ripplex1-send-xrp.js'></script>
+    <script src='ripplex2-send-currency.js'></script>
+    <script src='ripplex8-escrow.js'></script>
+  </head>
+  
+<!-- ************************************************************** -->
+<!-- ********************** The Form ****************************** -->
+<!-- ************************************************************** -->
+
+  <body>
+    <h1>Time-based Escrow Test Harness</h1>
+    <form id="theForm">
+      Choose your ledger instance:  
+      &nbsp;&nbsp;
+      <input type="radio" id="tn" name="server"
+        value="wss://s.altnet.rippletest.net:51233" checked>
+      <label for="tn">Testnet</label>
+      &nbsp;&nbsp;
+      <input type="radio" id="dn" name="server"
+        value="wss://s.devnet.rippletest.net:51233">
+      <label for="dn">Devnet</label>
+      <br/><br/>
+      <button type="button" onClick="getAccountsFromSeeds()">Get Accounts From Seeds</button>
+      <br/>
+      <textarea id="seeds" cols="40" rows= "2"></textarea>
+      <br/><br/>
+      <table>
+        <tr valign="top">
+          <td>
+            <table>
+              <tr valign="top">
+                <td>
+                <td>
```

Comment:
Seems like a duplicated/unclosed table cell here. Thankfully the effects on the page are minimal (it moves the whole thing a couple pixels over) because browsers are pretty good at working around typos like this.

```suggestion
                <td>
```